controllers/artefacts.rb

```diff
@@ -140,23 +154,28 @@ class ArtefactsController < ApplicationController
                     end
 
                     ontology, latest_submission = get_ontology_and_latest_submission
+                    check_access(ontology)
                     attributes, page, size = settings_params(model_class).first(3)
                     type = LinkedData::Models::Class.class_rdf_type(latest_submission)
                     if type.to_s == "http://www.w3.org/2004/02/skos/core#Concept"
                         reply handle_resources_request(ontology, latest_submission, model_class, attributes, page, size)    
                     else
-                        reply empty_page(page, size)
+                        reply empty_page
                     end
                     
                 end
             end
-          
-            private
-            
-            def empty_page(page, size)
-                Goo::Base::Page.new(page, size, 0, [])
+
+            get '/resource' do
```

Comment:
here it should be `get  "/" ` no ?

Reply:
no here is the route to call `/resources/resource?uri=...`